### PR TITLE
Deal10

### PR DESCRIPTION
**Pull request by nicola-giuliani**
@luca-heltai this version is compiling with deal.10.pre. What do you think? I have troubles running the indenter

### CODE REVIEW

**Comment by luca-heltai** [comment]
The indenter has been updated. I think I need to make sure that the docker image uses the new indenter...

File: CMakeLists.txt [hunk 1]
```diff
@@ -65,7 +65,7 @@ IF(POLICY CMP0037)
   CMAKE_POLICY(SET CMP0037 OLD)
 ENDIF()
 
-FIND_PACKAGE(deal.II 9.0 REQUIRED
+FIND_PACKAGE(deal.II 9.2 REQUIRED
```
Comment (luca-heltai) [inline CMakeLists.txt]:
```suggestion
FIND_PACKAGE(deal.II 10.0 REQUIRED
```

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-9.1.1
+9.2.0
```
Comment (luca-heltai) [inline VERSION]:
```suggestion
10.0.0-pre
```

**Review by luca-heltai — APPROVED** [review]
Since these are all incompatible changes, and require deal.II 10.0.0-pre, we should make sure this is what we ask for in the CMakeLists.txt, otherwise we break compatibility

File: CMakeLists.txt (hunk 1 above)
Comment (nicola-giuliani) [reply CMakeLists.txt]: absolutely

File: VERSION (hunk 2 above)
Comment (nicola-giuliani) [reply VERSION]: yep, done

**Comment by luca-heltai** [comment]
Thanks!

**Pull request merged**